### ISSUE TITLE
Very small read-me change

### ISSUE THREAD

**Issue by dginovker** [monero-project/kovri#964]
Hey guys sorry to bother :P I was just wondering if you could maybe update the readme. Right now it says:

> This page is also available in the following languages
> 
> - Italian
> - Spanish
> - Russian
> - French
> - German
> - Danishs

I think it would make more sense to change that to:

This page is also available in the following languages

- [Italiano](https://github.com/monero-project/kovri-docs/blob/master/i18n/it/README.md)
- Español (incompleto)
- Pусский (Неполный)
- [Français](https://github.com/monero-project/kovri-docs/blob/master/i18n/fr/README.md)
- Deutsch (unvollständig)
- Dansk (ufuldstændig)

--Source--
```This page is also available in the following languages

- [Italiano](https://github.com/monero-project/kovri-docs/blob/master/i18n/it/README.md)
- Español (incompleto)
- Pусский (Неполный)
- [Français](https://github.com/monero-project/kovri-docs/blob/master/i18n/fr/README.md)
- Deutsch (unvollständig)
- Dansk (ufuldstændig)
```

Sorry if this is wrong place! Just trying to help :+1: 

Edit: I can actually do this myself if I can figure out how - do I just fork the project, change the readme and make a pull request? I can do that =D

Edit2: Thanks for the comments. I've created a pull request! https://github.com/monero-project/kovri/pull/966

**Comment by anonimal** [comment]
>do I just fork the project, change the readme and make a pull request? I can do that =D

Yes, exactly. Thanks.

**Comment by erciccione** [comment]
I odn't really see too much utility in adding`incomplete`, on the side of not existent readmes, if there is no link it clearly doesn't exist, but i'm not against it if you guys think it could be useful.

**Comment by anonimal** [comment]
Resolved in #966.
